src/main/java/com/educare/unitylend/model/Community.java

```diff
@@ -4,14 +4,35 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
-import java.util.UUID;
-
 @Data
 @NoArgsConstructor
 @AllArgsConstructor
 public class Community {
-    private UUID communityId;
-    private String communityName;
+    private String communityid;
```

Comment:
As per the Java convention variable names should be in camelCase.
i.e (communityid should be communityId).

Reply:
It has been done.